example/next/utils/initSupabase.js

```diff
@@ -1,11 +1,13 @@
 import { GoTrueClient } from '@supabase/gotrue-js'
 
-const supabaseURL = 'http://localhost:9999'
+const supabaseURL = process.env.NEXT_PUBLIC_SUPABASE_URL
+const supabaseAnon = process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY
 
 const auth = new GoTrueClient({
-  url: `${supabaseURL}`,
+  url: `${supabaseURL}/auth/v1`,
```

Comment:
hmm why do we add the `/auth/v1` here?

Reply:
That's the auth endpoint.

Reply:
ah yeah i thought the `supabaseURL` included that already - it might lead to confusing behaviour if someone's using gotrue-js outside of supabase and trying out this next example but i'm ok with going ahead with this and changing it later if we need to since it's just an example